cmake/modules/toolconf/GCovR.cmake

```diff
@@ -14,23 +14,33 @@
 # SPDX-License-Identifier:	Apache 2.0
 # ______________________________________________________
 
-option(${HDK_ROOT_PROJECT_NAME_UPPER}_TOOLCONF_USE_GCOVR "Use gcovr in project" OFF)
+# GCovr will use gcov executable as default
```

Comment:
It should not use gcov by default. Instead, check which one of the coverage executables is enabled and use that. Also, enabling of both coverage executables at the same time, or enabling gcovr/lcov without enabling a coverage executable are error conditions.